test/acceptance/features/steps/steps.py

```diff
@@ -394,3 +397,40 @@ def etc_cluster_is_running(context, etcd_name):
         print("etcd cluster not present, creating etcd cluster")
         assert etcd_cluster.create() is True, "etcd cluster is not created"
         assert etcd_cluster.is_present() is True, "etcd cluster is not present"
+
+
+@when(u'Invalid Service Binding is applied')
+def invalid_sbr_is_applied(context):
+    sbr = ServiceBinding()
+    # Get resource version of sbr if sbr is available
+    if context.__contains__("sbr_name"):
```

Comment:
I can see many uses of the special method `__contains__`. Is there any problem to use `in` operator?

Ref. https://stackoverflow.com/questions/1964934/what-does-contains-do-what-can-call-contains-function

Reply:
I didn't really think about it `__contains__` was already in use so I simply continued it but will look into it and if `in` works will replace `__contains__` with `in`.

Reply:
Hey updated as you suggested. PTAL.